pytorch_lightning/accelerators/cpu.py

```diff
@@ -11,6 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from typing import Any, Dict, Union
```

Comment:
I would add a dummy implementtation of `get_device_stats` just so we don't raise a not implemented error. in the future we could use something like psutil to get per-process stats https://psutil.readthedocs.io/